.storybook/theme.ts

```diff
@@ -0,0 +1,23 @@
+import { ThemeVars, create } from "@storybook/theming";
+import logo from "./logo.svg";
+
+interface ExtendedThemeVars {
+  addonNotesTheme: any;
+}
+
+export default create({
+  base: "light",
+  brandTitle: "Calcite App Components",
+  brandUrl: "./",
+  brandImage: logo,
+  addonNotesTheme: {
+    code: {
+      whiteSpace: "normal !important"
```

Comment:
question: Whats up with these `!important` styles?

Reply:
I got this from `calcite-components`, but after a quick test it doesn't seem like anything is overriding the theme, so I think we can drop the `!important`s.

Reply:
💥